src/api/GoalsAPI/index.ts

```diff
@@ -0,0 +1,46 @@
+import { db, GoalItem } from '@models';
+import { getJustDate } from '@src/utils';
+
+export const resetDatabase = () => db.transaction('rw', db.goalItems, async () => {
+  await Promise.all(db.tables.map((table) => table.clear()));
+});
+
+export const addGoal = (goalDetails :{
+  title: string,
+  duration: string,
```

Comment:
Are you sure we want the duration to be a `string`?

Reply:
I missed it